src/factories/bittrexFactories.ts

```diff
@@ -45,9 +45,12 @@ export function getSubscribedFeeds(options: ExchangeFeedConfig, products: string
             return reject(new Error('TIMEOUT. Could not connect to Bittrex Feed server'));
         }, 30000);
         feed.on('websocket-connection', () => {
-            feed.subscribe(products);
-            clearTimeout(timeout);
-            return resolve(feed);
+            feed.subscribe(products).then(() => {
```

Comment:
Since we return a promise now, may as well use it properly :)

Reply:
indeed. coming from the land of beautiful async await py, I tend to miss things like this.